src/runtime/plugins/script.ts

```diff
@@ -48,6 +49,15 @@ export default defineNuxtPlugin(nuxtApp => {
       await this.loadTurnstile()
       return (window as any).turnstile.reset(element)
     },
+    async remove(id: string) {
+      if (process.server) return
+
+      if (addTurnstileScript.value) {
+        return (window as any).turnstile.remove(id)
+      } else {
+        console.warn('cannot remove a Turnstile widget without enabling Turnstile')
```

Comment:
```suggestion
        console.warn('Cannot remove a Turnstile widget without enabling Turnstile.')
```

Also, should this maybe use `consola` instead?

Reply:
sure :+1:

Reply:
@danielroe inside modules `@nuxt/kit`'s logger is used... should the same be used inside plugins or `consola` or `console`?

Reply:
I would use console.